Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Reflection.Metadata;
+using System.Runtime.CompilerServices;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static float IsNumber(string input)
+        {
+            var isEmpty = String.IsNullOrEmpty(input);
+           
+            if (isEmpty) return 0f;
+            
+            if ((input) == "0") return 0f;
+
+            var isNumber = float.TryParse(input, out var result);
+            if (!isNumber) 
+            {
+                Console.Write(@"""" + input + @""" " + "is not a valid number.");
+
+                return -1f;
+            }
+            
+            return result;
+        }
+        
+        public static string GetString(string message)
+        {
+            Console.WriteLine(message);
+            
+            var input = Console.ReadLine();
+
+            bool isEmpty = String.IsNullOrEmpty(input);
+            if (isEmpty)
+            {
+                Console.Write("Name cannot be empty.");
+
+                return "-";
+            }
+
+            return input;
+        }
+       
+        public static int GetInt(string message)
+        {
+            Console.WriteLine(message);
+            
+            string int1 = Console.ReadLine();
+            
+            var isInt = IsNumber(int1);
+            
+            int result = Convert.ToInt32(isInt);
+            
+            return (result);
+        }
+       
+        public static float GetFloat(string message)
+        {
+            Console.WriteLine(message);
+            
+            string float1 = Console.ReadLine();
+            
+            var isFloat = IsNumber(float1);
+           
+            return isFloat;
+        }
+
+        public static float Bmi(float weight, float height)
+        {
+            string heightError = ("Height cannot be equal or less than zero, but was " + height);
+            string weightError = ("Weight cannot be equal or less than zero, but was " + weight);
+            string failedCalc = ("Failed calculating BMI. Reason:");
+
+            bool isW0OrLess = ((weight) <= 0);
```

Comment:
Avoid naming variables with just 1 letter, unless it's a well known abbreviation. instead of 'w', please write "Weight".

Reply:
Makes code less readable otherwise and I was confused for a split second what it was for.